config/checkstyle-examples-checks.xml

```diff
@@ -112,9 +112,13 @@
     <!-- Regexp -->
     <module name="RegexpSinglelineJava">
       <property name="id" value="exampleClassName"/>
-      <property name="format" value="^class Example\d+"/>
+      <property name="format" value="class Example\d+"/>
       <property name="minimum" value="1"/>
       <property name="maximum" value="1"/>
+      <message key="regexp.minimum"
+               value="Example test input must declare exactly one class matching ''Example\d+''"/>
+      <message key="regexp.maximum"
+               value="Example test input must declare exactly one class matching ''Example\d+''"/>
```

Comment:
We needed to be a bit more lenient (allowing for `final class`, etc.), and make it clear: there should be exactly one class in each input with such naming.